dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -168,5 +170,72 @@ public void ValidateGetSetBackground()
                 Verify.AreSame(blueBrush, repeater.Background);
             });
         }
+
+        [TestMethod]
+        public void NestedRepeaterWithDataTemplateScenario()
```

Comment:
There aren't any comments in this function and the name is pretty generic sounding. I wasn't sure what it was supposed to... do.